### PR TITLE
Fix utm coordinate print

### PR DESCRIPTION
**Pull request by floroeske**

### CODE REVIEW

File: src/navsat_transform.cpp [hunk 1]
```diff
@@ -754,9 +754,9 @@ void NavSatTransform::setTransformGps(
   utm_meridian_convergence_ *= navsat_conversions::RADIANS_PER_DEGREE;
 
   RCLCPP_INFO(
-    this->get_logger(), "Datum (latitude, longitude, altitude) is (%d, %d, %d)",
+    this->get_logger(), "Datum (latitude, longitude, altitude) is (%f, %f, %f)",
```
Comment (SteveMacenski) [inline src/navsat_transform.cpp]: Can you limit the decimals please (`%0.2f`)

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Looks fine, lets just make the printing look reasonable

File: src/navsat_transform.cpp (hunk 1 above)
Comment (floroeske) [reply src/navsat_transform.cpp]: Yep, thanks. Done.

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
@floroeske the PR is failing due to linter errors. Make sure to run ament_uncrustify and ament_cpplint

**Comment by SteveMacenski** [comment]
Thanks!

**Pull request merged**